3rdparty/stblib/LICENSE

```diff
@@ -0,0 +1,37 @@
+This software is available under 2 licenses -- choose whichever you prefer.
```

Comment:
It feels kinda weird to list both licenses in a downstream repository like this, I'm guessing we're actually supposed to pick one. We can just go with Alternative A.

Reply:
okeydoke